airflow/utils/log/logging_mixin.py

```diff
@@ -58,6 +58,20 @@ def _set_context(self, context):
             set_context(self.log, context)
 
 
+class RemoteLoggingMixin:
+    """
+    Define a logging handler based on a remote service (e.g. ELK, StackDriver). Subclasses must override
+    REMOTE_LOG_NAME and get_remote_log_url().
+    """
+    REMOTE_LOG_NAME: str = 'RemoteLog'
```

Comment:
```suggestion
    def log_name(self):
        raise NotImplementedError('Remote logger must provide remote name')
```
This will force the user to configure the name.